solutions/core-landing-zone/mgmt-project/project-sink.yaml

```diff
@@ -14,6 +14,9 @@
 ######
 # Project sink for the Platform and Component logs of the Landing Zone Management Cluster
 # Destination: Cloud Logging bucket hosted inside logging project
+# AU-3, AU-3(1) - Sink defined at folder that will allow all the projects underneath the folder to send the logs to the logging bucket in the logging project
+# AU-4(1), AU-6(4), AU-9(2) - Log sinks sending the logs to same project in same region having a logging bucket
+# AU-12, AU-12(1) - Log Sinks defined in Log Router check each log entry against the inclusion filter and exclusion filter that determine which destinations that the log entry is sent to
```

Comment:
Inclusion/exclusion filters define the scope of the logs sent to a particular sink (ie: >= INFO)